examples/cart/index.js

```diff
@@ -97,7 +97,9 @@ $(function() {
   /* Update product image based on selected variant
   ============================================================ */
   function updateVariantImage(image) {
-    $('#buy-button-1 .variant-image').attr('src', image.src);
+    var src = typeof image === 'undefined' ? cart.noImageURI : image.src;
```

Comment:
This will break if we switch it up to null. Why not just `src = (image) ? image.src : IMAGE_URI`